mmv1/products/firebase/api.yaml

```diff
@@ -57,16 +57,77 @@ objects:
           'https://firebase.google.com/'
       api: 'https://firebase.google.com/docs/projects/api/reference/rest/v1beta1/projects'
     properties:
-      - !ruby/object:Api::Type::String
+      - !ruby/object:Api::Type::Integer
         name: projectNumber
         output: true
         description: |
-          The number of the google project that firebase is enabled on.
+          Immutable. The globally unique, Google-assigned canonical identifier
+          for the Project. Use this identifier when configuring integrations and/or
+          making API calls to Firebase or third-party services.
+      - !ruby/object:Api::Type::String
+        name: projecId
+        output: true
+        description: |
+          Immutable. A user-assigned unique identifier for the Project.
+          This identifier may appear in URLs or names for some Firebase resources
+          associated with the Project, but it should generally be treated as a convenience
+          alias to reference the Project.
       - !ruby/object:Api::Type::String
         name: displayName
         output: true
         description: |
-          The GCP project display name
+          The user-assigned display name of the Project. This field may only be assigned on Update.
+      - !ruby/object:Api::Type::KeyValuePairs
+        name: annotations
+        description: |
+          Set of user-defined annotations for the FirebaseProject. These annotations
+          are intended solely for developers and client-side tools Firebase services
+          will not mutate this annotation set.  This field may only be assigned on Update.
+      - !ruby/object:Api::Type::NestedObject
+        name: resources
```

Comment:
default resources are nearly obsolete.  Lets omit this one unless there is a specific customer issue asking for it.

If we do keep this, our description will need to explicitly call out this is going to be deprecated.